src/main/java/io/lumigo/core/instrumentation/agent/Loader.java

```diff
@@ -19,28 +22,12 @@ public static void instrument(java.lang.instrument.Instrumentation inst) {
                 new ApacheKafkaProducerInstrumentation();
         ApacheKafkaConsumerInstrumentation apacheKafkaConsumerInstrumentation =
                 new ApacheKafkaConsumerInstrumentation();
+        AwsLambdaRequestHandlerInstrumentation awsLambdaRequestHandlerInstrumentation =
+                new AwsLambdaRequestHandlerInstrumentation();
         AgentBuilder builder =
                 new AgentBuilder.Default()
                         .disableClassFormatChanges()
                         .with(AgentBuilder.RedefinitionStrategy.RETRANSFORMATION)
-                        .ignore(
```

Comment:
We have to remove this optimization because we dont know the handler entry point